primedev/plugins/pluginmanager.cpp

```diff
@@ -182,7 +182,28 @@ void ConCommand_reload_plugins(const CCommand& args)
 	g_pPluginManager->ReloadPlugins();
 }
 
+void ConCommand_unload_plugins(const CCommand& args)
+{
+	for (const Plugin& plugin : g_pPluginManager->GetLoadedPlugins() | std::views::reverse)
+	{
+		std::string name = plugin.GetName();
+
+		if (args.ArgC() >= 2 && !strcmp(name.c_str(), args.Arg(1)))
+			continue;
+
+		if (plugin.Unload())
```

Comment:
Might be wise to print if unloading fails, wdyt?

Reply:
yeah it be wise to add a log call there